### ISSUE TITLE
guided_merge

### ISSUE THREAD

**Issue by drtamermansour** [snipe-bio/snipe#29]
The pull request of '**guided_merge**' is using the command sometimes as `guided_merge`  and sometimes as `guided-merge`

**Comment by drtamermansour** [comment]
Also, I suggest the final output statement of the command to be more informative e.g.,  report the number of merged signatures

current:
Merged signature for experiment 'ERX3279744' exported to merged_sigs/ERX3279744.zip


suggested:
Renamed the signature of run "run file basename" to  merged_sigs/ERX3279744.zip
Merged 3 run signatures into merged_sigs/ERX3279744.zip

**Comment by mr-eyes** [comment]
there is no `guided_merge` command, it's only `guided-merge` but I will make sure I unify it across the non-code part.

> The pull request of '**guided_merge**' is using the command sometimes as `guided_merge` and sometimes as `guided-merge`

**Comment by mr-eyes** [comment]
> Also, I suggest the final output statement of the command to be more informative e.g., report the number of merged signatures
> 
> current: Merged signature for experiment 'ERX3279744' exported to merged_sigs/ERX3279744.zip
> 
> suggested: Renamed the signature of run "run file basename" to merged_sigs/ERX3279744.zip Merged 3 run signatures into merged_sigs/ERX3279744.zip

When you start running it that will be the output. But at the end of the merging you will see this report:

```
Guided Merge Summary:
Total signatures mapped in table: 41387
Valid signatures processed: 41387
Invalid or missing signatures skipped: 0
Total experiments merged: 15628
 - Experiment 'ERX000020': 15 signatures merged into ../experiment_sigs/ERX000020.zip
 - Experiment 'ERX000021': 13 signatures merged into ../experiment_sigs/ERX000021.zip
 - Experiment 'ERX000022': 15 signatures merged into ../experiment_sigs/ERX000022.zip
 - Experiment 'ERX000024': 14 signatures merged into ../experiment_sigs/ERX000024.zip
```

And also this:
```
Error: Failed to merge signatures for experiment 'SRX147820': All provided signatures are empty.
Error: Failed to merge signatures for experiment 'SRX147821': All provided signatures are empty.
Error: Failed to merge signatures for experiment 'SRX147822': All provided signatures are empty.
Error: Failed to merge signatures for experiment 'SRX147823': All provided signatures are empty.
Merged signature for experiment 'SRX154766' exported to ../experiment_sigs/SRX154766.zip
Merged signature for experiment 'SRX191928' exported to ../experiment_sigs/SRX191928.zip
Merged signature for experiment 'SRX192147' exported to ../experiment_sigs/SRX192147.zip
Merged signature for experiment 'SRX192150' exported to ../experiment_sigs/SRX192150.zip
Merged signature for experiment 'SRX192151' exported to ../experiment_sigs/SRX192151.zip
```

But it's very verbose; I will try to make a writable report.

**Comment by drtamermansour** [comment]
> there is no `guided_merge` command, it's only `guided-merge` but I will make sure I unify it across the non-code part.
> 
> > The pull request of '**guided_merge**' is using the command sometimes as `guided_merge` and sometimes as `guided-merge`


I am referring to this  pull request:
https://github.com/snipe-bio/snipe/pull/18?fbclid=IwY2xjawGIxlxleHRuA2FlbQIxMAABHR1uyAQtLyt7UeTOv-JwaJ_jADGWaufhpOWXnHa-fWlzUVQwc--cNSD0WQ_aem_s6qZd8IKCjRx4aTycAAe2w

Search for `guided_merge`. You will find multiple matches including the example code of the cpommand.

**Comment by mr-eyes** [comment]
Resolved in #31
